extensions/2.0/Vendor/EXT_materials_bump/README.md

```diff
@@ -0,0 +1,142 @@
+# EXT\_materials\_bump
+
+## Contributors
+
+- Ben Houston, Threekit [@bhouston](https://github.com/bhouston)
+
+Copyright 2023 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft, Ratified by the Khronos Group
```

Comment:
An extension cannot be ratified and draft at the same time.

Reply:
fixed!